pkg/operator/config.go

```diff
@@ -58,6 +60,55 @@ type ContainerConfig struct {
 	EnableProbes  bool
 }
 
+type Annotations struct {
+	AnnotationsString string
+	AnnotationsMap    map[string]string
+}
+
+// Implement the flag.Value interface
```

Comment:
These could probably be merged with labels functions with some sort of interfaces / generics. Apologies, not that experienced ;)